### PR TITLE
Feat: added SDKkey and environmentKey in datafile

### PR DESCRIPTION
**Pull request by yasirfolio3**
## Summary
-  Added support for `sdkKey` and `environmentKey` keys in optimizely config.

## Test plan
1. All unit tests should pass.
2. FSC tests should pass for PR: https://github.com/optimizely/fullstack-sdk-compatibility-suite/pull/513

FSC Result:
https://travis-ci.com/github/optimizely/fullstack-sdk-compatibility-suite/builds/227236938

### CODE REVIEW

**Review by keppel2 — APPROVED** [review]

File: pkg/config/datafileprojectconfig/config.go [hunk 1]
```diff
@@ -80,11 +82,21 @@ func (c DatafileProjectConfig) GetAttributeID(key string) string {
 	return c.attributeKeyToIDMap[key]
 }
 
-// GetBotFiltering returns GetBotFiltering
+// GetBotFiltering returns botFiltering
 func (c DatafileProjectConfig) GetBotFiltering() bool {
 	return c.botFiltering
 }
 
+// GetSdkKey returns sdkKey
```
Comment (msohailhussain) [inline pkg/config/datafileprojectconfig/config.go:90]: sdkKey for specific environment.

File: pkg/config/datafileprojectconfig/config.go [hunk 2]
```diff
@@ -80,11 +82,21 @@ func (c DatafileProjectConfig) GetAttributeID(key string) string {
...
+func (c DatafileProjectConfig) GetSdkKey() string {
+	return c.sdkKey
+}
+
+// GetEnvironmentKey returns environmentKey
```
Comment (msohailhussain) [inline pkg/config/datafileprojectconfig/config.go:95]: current environment of the datafile.

File: test-data/decide-test-datafile.json [hunk 3]
```diff
@@ -342,5 +342,7 @@
       "key": "event_multiple_running_exp_attached"
     }
   ],
-  "revision": "241"
+  "revision": "241",
+  "sdkKey": "ValidProjectConfigV4",
+  "environmentKey": "production"
```
Comment (msohailhussain) [inline test-data/decide-test-datafile.json:347]: eol

**Review by msohailhussain — APPROVED** [review]
lgtm

**Pull request merged**